frontend/dev.sh

```diff
@@ -24,8 +24,8 @@ cd openchat-shared
 pm2 start --name "shared" "npm run dev" 
 cd ../openchat-agent
 pm2 start --name "agent" "npm run dev" 
-cd ../openchat-push
-pm2 start --name "push" "npm run dev" 
+# cd ../openchat-push
```

Comment:
Do you want this included?

Reply:
yes it only affects dev and it just puts a bit less strain on my machine